example/javascriptlib/basic/3-custom-build-logic/build.mill

```diff
@@ -9,15 +9,15 @@ object foo extends TypeScriptModule {
     allSources().map(f => os.read.lines(f.path).size).sum
   }
 
-  /** Generate resources using lineCount of sources */
-  def resources = Task {
+  /** Generate lineResource using lineCount of sources */
+  def lineResource = Task {
```

Comment:
Why are we renaming this `lineResources`? We should use the `resources` logic you implemented, which would remove the need for the custom `mkENV` override